docs/setup/security/shiro_authentication.md

```diff
@@ -247,6 +247,52 @@ knoxJwtRealm.principalMapping = principal.mapping
 authc = org.apache.zeppelin.realm.jwt.KnoxAuthenticationFilter
 ```
 
+### HTTP SPNEGO Authentication
+HTTP SPNEGO (Simple and Protected GSS-API NEGOtiation) is the standard way to support Kerberos Ticket based user authentication for Web Services. Based on [Apache Hadoop Auth](https://hadoop.apache.org/docs/current/hadoop-auth/index.html), Zeppelin supports ability to authenticate users by accepting and validating their Kerberos Ticket.
+
+When HTTP SPNEGO Authentication is enabled for Zeppelin, the [Apache Hadoop Groups Mapping](https://hadoop.apache.org/docs/r2.8.0/hadoop-project-dist/hadoop-common/GroupsMapping.html) configuration will used internally to determine group membership of user who is tyring to log in. Role-based access permission can be set based on groups as seen by Hadoop.
```

Comment:
why r2.8.0?

Reply:
Because r2.8.0 has better documentation and r2.7.3 doesn't have any GroupsMapping even though the Group mapping has been available since long. If it helps, I can use this doc link from r2.7.3 (but this is not as good as r2.8.0 one).
https://hadoop.apache.org/docs/r2.7.3/hadoop-project-dist/hadoop-hdfs/HdfsPermissionsGuide.html#Group_Mapping

Reply:
@felixcheung Do let me know if we should be sticking to 2.7.3 doc link. Thanks.